databases/week2/part2.sql

```diff
@@ -0,0 +1,30 @@
+-- Create a new database
+CREATE DATABASE school;
+
+-- Switch to the new database
+USE school;
+
+-- Create the Class table
+CREATE TABLE Class (
+  id INT PRIMARY KEY,
```

Comment:
I'd suggest you use the AUTO_INCREMENT option for defining ids if there are no other requirements because it allows you to avoid specifying an id each time you insert new values and ensures that your ids are unique.